test/common_utils.py

```diff
@@ -131,6 +146,121 @@ def accept_output(update_type):
         else:
             self.assertNestedTensorObjectsEqual(output, expected, rtol=rtol, atol=atol)
 
+    def assertEqual(self, x, y, prec=None, message='', allow_inf=False):
```

Comment:
This all copied from PyTorch's `common_utils`

Reply:
this is maybe more general than it needs to be - would `assertNestedTensorObjectsEqual` suffice ? this function e.g. does coercion between numbers and tensor results that you wouldn't want to allow for testing model equality. it's also a good amount of code.

Reply:
`assertEqual` has been pretty stable and widely used in PyTorch, I think it's better to just copy the entire thing and avoid having to move it over piece by piece later on if some functionality is missing from `assertNestedTensorObjectsEqual`

Reply:
okay, but we shouldn't have both in this file, we should only have one.